src/App.jsx

```diff
@@ -3,14 +3,21 @@ import React from 'react';
 import Todo from './Components/Todo';
 import Header from './Components/Header'
 import Footer from './Components/Footer'
+import SettingsForm from './Components/SettingsForm';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
 export default class App extends React.Component {
   render() {
     return (
       <>
+      <BrowserRouter>
       <Header />
-      <Todo />
+      <Routes>
+      <Route path="/" element={<Todo />} />
+        <Route path="/settings" element={<SettingsForm />} />
+        </Routes>
```

Comment:
One thing to be careful of: State only lasts as long as its component is being rendered. When you go to change your settings, the `<Todo />` component is no longer rendered, so when you go back, all the items in your `list` are gone. You could fix this by moving your `list` to context, but I'll say don't worry about it for now, since today's lab is pulling that list from an API.